convene-web/app/models/guide.rb

```diff
@@ -21,7 +21,7 @@ def persisted?
   # TODO: There is probably a way to get an "ActiveRecord::Query"-like
   # interface without writing it ourselves.
   def self.all
-    %i[locked_rooms].lazy.map do |slug|
+    %i[neighborhoods people spaces rooms furniture getting_around identification].lazy.map do |slug|
```

Comment:
Is `lazy` useful here since we are going to evaluating all values in the new block?

Reply:
After reading https://ruby-doc.org/core-2.5.0/Enumerator/Lazy.html, I now understand what it does.
I think you were thinking ahead where this array could be really large someday.

Reply:
I expect that I'll actually wind up moving this "feature" to a piece of Furniture; so that a Room can have a GuideBook composed of a bunch of markdown files.